tools/zetasql_helper/java/com/google/bigquery/utils/zetasqlhelper/Client.java

```diff
@@ -0,0 +1,40 @@
+package com.google.bigquery.utils.zetasqlhelper;
+
+import io.grpc.Channel;
+
+/**
+ * Client to establish connections with ZetaSQL Helper server.
+ */
+public class Client {
+
+    private static ZetaSqlHelperLocalServiceGrpc.ZetaSqlHelperLocalServiceBlockingStub stub;
+    private static int port = 50051;
```

Comment:
Can we use random port here?

Reply:
The default one is 50051, but users can always set their own port using setPort method